docs/src/pages/docs/agents/01-agent-memory.mdx

```diff
@@ -66,13 +67,15 @@ const response1 = await agent.generate("What were we discussing earlier?", {
 const response2 = await agent.generate("What were we discussing earlier?", {
   resourceId: "user_123",
   threadId: "thread_456",
-  thread: {
-    injectRecentMessages: 5,  // Only inject 5 recent messages
-    injectVectorHistorySearch: {
-      includeResults: 2,      // Only get 2 semantic search results
-      includePrevious: 1,     // Include 1 message before each result
-      includeNext: 1,         // Include 1 message after each result
-    }
+  memoryOptions: {
+    recentMessages: 5,  // Only inject 5 recent messages
+    historySearch: {
+      topK: 2,         // Only get 2 semantic search results
+      messageRange: {   // Context around each result
```

Comment:
@TylerBarnes I would just show `messageRange: 2` in this example and the previous one

Reply:
We didn't talk about this, but I assume this is also an optional key